doc/install/index.rst

```diff
@@ -1,72 +1,108 @@
-=======================
- Installation (Manual)
-=======================
+.. _install-overview:
 
+===============
+Installing Ceph
+===============
 
-Get Software
-============
+There are various options for installing Ceph. Review the documention for each method before choosing the one that best serves your needs.
 
-There are several methods for getting Ceph software. The easiest and most common
-method is to `get packages`_ by adding repositories for use with package
-management tools such as the Advanced Package Tool (APT) or Yellowdog Updater,
-Modified (YUM). You may also retrieve pre-compiled packages from the Ceph
-repository. Finally, you can retrieve tarballs or clone the Ceph source code
-repository and build Ceph yourself.
+We recommend the following installation methods:
 
+   * cephadm
+   * Rook
 
-.. toctree::
-   :maxdepth: 1
 
-	Get Packages <get-packages>
-	Get Tarballs <get-tarballs>
-	Clone Source <clone-source>
-	Build Ceph <build-ceph>
-    	Ceph Mirrors <mirrors>
-	Ceph Containers <containers>
+We offer these other methods of installation in addition to the ones we recommend:
 
+   * ceph-ansible
+   * ceph-deploy (no longer actively maintained)
+   * Deepsea (Salt)
+   * Juju
+   * Manual installation (using packages)
+   * Puppet
 
-Install Software
-================
 
-Once you have the Ceph software (or added repositories), installing the software
-is easy. To install packages on each :term:`Ceph Node` in your cluster. You may
-use  ``ceph-deploy`` to install Ceph for your storage cluster, or use package
-management tools. You should install Yum Priorities for RHEL/CentOS and other
-distributions that use Yum if you intend to install the Ceph Object Gateway or
-QEMU.
+Recommended Methods of Ceph Installation
+========================================
 
-.. toctree::
-   :maxdepth: 1
+cephadm
+-------
 
-	Install ceph-deploy <install-ceph-deploy>
-    	Install Ceph Storage Cluster <install-storage-cluster>
-	Install Ceph Object Gateway <install-ceph-gateway>
-	Install Virtualization for Block <install-vm-cloud>
+Installs Ceph using containers and systemd.
 
+* :ref:`cephadm-bootstrap`
 
-Deploy a Cluster Manually
-=========================
+     * cephadm is supported only on Octopus and newer releases.
+     * cephadm is fully integrated with the new orcehstration API and fully supports the new CLI and dashboard features to manage cluster deployment.
```

Comment:
typo

Reply:
note to future Zac: s/orcehstration/orchestration/